packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/responses.ts

```diff
@@ -0,0 +1,195 @@
+import { RDD } from '@chainlink/gauntlet-terra'
+import { InspectInstruction, instructionToInspectCommand } from '../../../abstract/inspectionWrapper'
+import { CONTRACT_LIST } from '../../../../lib/contracts'
+import { CATEGORIES } from '../../../../lib/constants'
+import { LCDClient } from '@terra-money/terra.js'
+import { getLatestOCRNewTransmissionEvent, parseObserversByLength, RoundData } from '../../../../lib/inspection'
+import { inspection, logger } from '@chainlink/gauntlet-core/dist/utils'
+
+type CommandInput = {
+  transmitters: string[]
+  description: string
+  observersInfo: { [address: string]: ObserverInfo }
```

Comment:
We are splitting the oracle information between `transmitters` and `observers`, but at this point, we don't know who are transmitters and who are the observers. Let's just have one `oracles` that contains all the info necessary:
```js
type CommandInput = {
  transmitters: string[]
  description: string
  oracles: Oracle[]
}

type Oracle = {
  transmitter: string
  name: string
  apis: string
}
```